AloisBlue/Food-Fast-Food
# #160191044 Login Page

#### What does this PR do?
It creates the login page of my app
#### Description of Task to be completed?
It should help users to login in to their account
#### How should this be manually tested?
Clone it by git clone https://github.com/AloisBlue/Food-Fast-Food.git
cd into app main folder and access the folder with html files to see login.html
#### What are the relevant pivotal tracker stories?
#160191044
#### Screenshots 
![screenshot from 2018-09-22 00-58-35](https://user-images.githubusercontent.com/37014066/45913555-a909bd80-be02-11e8-9785-2c1b264f5f50.png)
